tokio-postgres/Cargo.toml

```diff
@@ -55,6 +55,14 @@ pin-project-lite = "0.2"
 phf = "0.11"
 postgres-protocol = { version = "0.6.5", path = "../postgres-protocol" }
 postgres-types = { version = "0.2.4", path = "../postgres-types" }
+tokio = { version = "1.27", default-features = false }
```

Comment:
Tokio doesn't have any default features, so I don't think this dependency tweaking is necessary.